piqueserver/scripts/savemap.py

```diff
@@ -18,32 +19,52 @@
 from twisted.logger  import Logger
 from pyspades.vxl import VXLData
 from piqueserver.config import config
+from piqueserver.commands import command
 
+savemap_config = config.section('savemap')
 log = Logger()
 
-savemap_config = config.section('savemap')
-LOAD_SAVED_MAP_OPTION = savemap_config.option('load_saved_map', False)
+@command('save', admin_only=True)
+def save(connection):
+    connection.protocol.save_map()
+
+@command('toggle_load_saved_map', admin_only=True)
+def toggle_load_saved_map(connection):
+    protocol = connection.protocol
+    protocol.LOAD_SAVED_MAP_OPTION = not protocol.LOAD_SAVED_MAP_OPTION
+    log.info("LOAD_SAVED_MAP: %s" % protocol.LOAD_SAVED_MAP_OPTION)
```

Comment:
This is wouldn't be a useful log message, especially for `info` level. Players using it also don't get any feedback what they set the value to because it doesn't return anything.